examples/reference.yml

```diff
@@ -15,6 +15,10 @@ server:  # Server settings.
   # TLS certificate and key.
   certificate: "/path/to/server.pem"
   key: "/path/to/server.key"
+  # Alternately, you may use LetsEncrypt
```

Comment:
to clarify - in this case, using separate token.{certificate,key} is a must, yes?